### PR TITLE
[doc][ybm] Tablet peer alert

### PR DESCRIPTION
**Pull request by ddhodge**
Tablet peer alert

@netlify /preview/yugabyte-cloud/cloud-monitor/cloud-alerts/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md [hunk 1]
```diff
@@ -93,6 +94,17 @@ If you get frequent cluster alerts on a [Sandbox cluster](../../cloud-basics/cre
 
 {{< /note >}}
 
+#### Fix tablet peer alerts
+
+YugabyteDB Aeon sends a notification when the number of [tablet peers](../../../architecture/docdb-replication/replication/#tablet-peers) in the cluster exceeds the threshold, as follows:
+
+- Number of tablet peers is 80% of the cluster limit (Warning).
```
Comment (daniel-yb) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md:101]:
```suggestion
- Number of tablet peers is 85% of the cluster limit (Warning).
```

**Review by daniel-yb — APPROVED** [review]
This looks good to me for now. Once we introduce guardrails that limit the count to 100%, we'll need to expand on this to explain that the DB will _reject_ new table creations and prevent tablet splitting until the cluster is scaled up (that work is ~2 weeks or so away)

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ YugabyteDB monitors the health of your clusters based on [cluster alert](#cluste
...
-| Needs Attention | [Node free storage](#fix-storage-alerts)<br/>[More than 34% of nodes down](#fix-nodes-reporting-as-down-alerts)<br/>[Memory Utilization](#fix-memory-alerts)<br/>[YSQL Connections](#fix-ysql-connection-alerts)<br/>[CPU Utilization](#fix-cpu-alerts) | Warning or Severe<br/>Warning or Severe<br/>Warning or Severe<br/>Warning<br/>Warning or Severe<br/>Warning or Severe<br/>Warning or Severe
+| Needs Attention | [Tablet peers](#fix-storage-alerts)<br/>[Node free storage](#fix-storage-alerts)<br/>[More than 34% of nodes down](#fix-nodes-reporting-as-down-alerts)<br/>[Memory Utilization](#fix-memory-alerts)<br/>[YSQL Connections](#fix-ysql-connection-alerts)<br/>[CPU Utilization](#fix-cpu-alerts) | Warning or Severe<br/>Warning or Severe<br/>Warning<br/>Warning or Severe<br/>Warning or Severe<br/>Warning or Severe |
 | Unhealthy | [More than 66% of nodes down](#fix-nodes-reporting-as-down-alerts)<br/>[CMK unavailable](#fix-cmk-unavailable-alerts)  | Severe<br/>Warning |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md:57]: CMK unavailable -> CMK Unavailable

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Comment by mdbridge** [comment]
the only quibble I have is "or vertically by adding vCPUs": this works but it's kind of misleading.

What matters is increasing the amount of RAM; however, on YBM fixed ratio is maintained between CPUs and RAM so increasing the CPUs will also increase the RAM.

**Comment by mdbridge** [comment]
the less likely option if you're running out of tablet peers is to just drop a bunch of tables to free up some tablet peers.
I feel like few customers would do that other than maybe the cases where people have universes with way too much stuff on them

 if we do point that out, we should make it clear that the tablets take some time to be freed up on the table is dropped; depends on PITR retention but could be a day

**Pull request merged**